packages/tscc/src/spec/TsccSpecWithTS.ts

```diff
@@ -1,7 +1,24 @@
-import {IInputTsccSpecJSON, ITsccSpecJSON, primitives, TsccSpec, TsccSpecError} from '@tscc/tscc-spec';
+import path = require('path');
 import * as ts from 'typescript';
+
+/** Closure Compiler spec imports. */
 import ITsccSpecWithTS from './ITsccSpecWithTS';
-import path = require('path');
+import {
+	IInputTsccSpecJSON,
+	ITsccSpecJSON,
+	TsccSpec,
+	TsccSpecError
+} from '@tscc/tscc-spec';
+
+/** Closure Compiler spec imports. */
+import {
+	ClosureCompilerFlags,
+	DEFAULT_FLAGS,
+	ES_TARGETS,
+} from './ClosureCompilerSpec';
+
+/** Set up the Closure Compiler flags. */
+const compilerFlags = { ...DEFAULT_FLAGS };
```

Comment:
If you are returning a global variable from `TsccSpecWithTS#getDefaultFlags`, it may be problematic when the same node process invokes tscc several times. This information is local to the `TsccSpecWithTS` class, so it should be a local variable in that method, if you want to keep `DEFAULT_FLAGS` variable in the other file.